extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -162,9 +186,26 @@ class AmpIframe extends AMP.BaseElement {
         this.updateHeight_(newHeight);
       }
     });
+    if (this.isClickToPlay_) {
+      listen(iframe, 'embed-clickready', () => {
```

Comment:
Done. 
But it would be easy to  use one pattern -my vote for using arrows everywhere and get rid of bind